.circleci/config.yml

```diff
@@ -119,27 +119,93 @@ jobs:
           name: Browser tests
           command: yarn test:browser-ci
 
+  release:
+    working_directory: ~/algoliasearch-client-javascript
+    description: Perform a new release of the JavaScript client
+    docker:
+      - image: circleci/node:14
+    steps:
+      - checkout
+      - run:
+          command: yarn install
+      - run:
+          command: |
+            if [[ -z "$GITHUB_TOKEN" ]]; then echo '$GITHUB_TOKEN is not set'; exit 1; fi
+            yarn shipjs trigger
+
 workflows:
   version: 2
   ci:
     jobs:
       - test_unit:
           version: '8'
+          name: 'test_unit_8'
+          filters:
```

Comment:
I think most of these filters can also be removed

Reply:
https://github.com/algolia/algoliasearch-client-javascript/pull/1210/commits/3198d231ec0bb705f8d1d4f1b2df95aebf87d2e9